packages/GA4Client/test/src/tests.js

```diff
@@ -860,8 +860,8 @@ describe('Google Analytics 4 Event', function () {
                     },
                 ];
 
-                window.dataLayer[0].should.eql(promotionResult1);
-                window.dataLayer[1].should.eql(promotionResult2);
+                window.dataLayer[0].should.match(promotionResult1);
+                window.dataLayer[1].should.match(promotionResult2);
```

Comment:
ran this locally, look like this can remain `eql` to remove unnecessary edits to this file